.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Use this template to report any issues or bugs you encounter while using our SDK.
+title: ''
+labels: 'bug report'
+assignees: ''
+
+---
+
+## Description
+Please provide a clear and concise description of the bug you're experiencing. Include details on what you expected to happen versus what actually occurred.
+
+## Steps to Reproduce
+1. List all the steps needed to reproduce the issue. Include any specific settings, configurations, or conditions.
+
+2. If possible, provide screenshots or a screen recording that clearly illustrates the issue.
+
+## Logs and Crash Reports
+1. Include any relevant logs.
+2. Enable verbose logging and attach the logs here.
+    - For SDK version `V5.*.*`:
+        - Enable by calling `com.adyen.checkout.core.AdyenLogger.setLogLevel(Log.VERBOSE)` in your application class.
+        - Filter your logs by `CO.` to see the checkout logs only.
+
+    - For SDK version `V4.*.*`:
+        - Enable by calling `com.adyen.checkout.core.log.Logger.setLogcatLevel(Log.VERBOSE)` in your application class.
+        - Filter your logs by `CO.` to see the checkout logs only.
+
+3. If the issue results in a crash, please include the full stack trace.
+
```

Comment:
I see a format from time to time with logs inside collapsable section, I thought that it made the whole issue easier to read.

This is how it would look like:
<details>
  <summary>Relevant logs</summary>

  ```
Insert your logs here
  ```
</details>

And this is the code for it:
````
<details>
  <summary>Relevant logs</summary>

  ```
Insert your logs here.
  ```
</details>
````

Feel free to add it if you like it.